### ISSUE TITLE
Wrong bootstrap command for Debian Stretch

### ISSUE THREAD

**Issue by thomasgoirand** [fraenki/puppet-galera#103]
Hi,

At the end of init.pp, the bootstrap_galera_cluster initialize the cluster with the command defined in params.pp. The issue is, under Debian Stretch, we get:

$bootstrap_command = 'service mysql start --wsrep_cluster_address=gcomm://'

when really, it should be:

$bootstrap_command = '/usr/bin/galera_new_cluster'

After fixing params.pp, it worked for me with the default mariadb-server and galera-3 packages. FYI, here's the way I called the galera class:

  class { 'galera':
    galera_servers      => $all_masters_ip,
    galera_master       => $first_master,
    mysql_package_name  => 'mariadb-server',
    vendor_type         => 'mariadb',
    root_password       => 'myPaSs',
    status_password     => 'myPaSs2',
    configure_repo      => false,
    configure_firewall  => false,
    galera_package_name => 'galera-3',
  }

which worked if fixing $bootstrap_command, but didn't by default.

**Comment by gregoryo2014** [comment]
This is true of Ubuntu as well. I worked around it with this code in my profile:

```
Exec <| title == 'bootstrap_galera_cluster' |> {
  command  => '/usr/bin/mysqld_bootstrap'
}
```

**Comment by fraenki** [comment]
@gregoryo2014 Thanks for these useful details. Care to submit a pull-request? :)

**Comment by gregoryo2014** [comment]
Sure thing https://github.com/fraenki/puppet-galera/pull/105

**Comment by gregoryo2014** [comment]
A new minor version will be handy - I'm working off a forked version and I'd like to stop doing that!
